artemis-server/src/main/resources/schema/XInclude.xsd

```diff
@@ -0,0 +1,49 @@
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
```

Comment:
There is no need for this, we already support xincludes.

Reply:
Referencing {{xi:include}} in the XSD follows the same pattern established for the {{xml}} namespace for the {{xml:specialAttrs}} attribute group. See my comment on the {{xi:include}} element reference.